Neos.Neos/Classes/Command/WorkspaceCommandController.php

```diff
@@ -178,13 +175,22 @@ public function createCommand(
         string $contentRepository = 'default'
     ): void {
         $contentRepositoryId = ContentRepositoryId::fromString($contentRepository);
+
+        $this->workspaceService->createPersonalWorkspace(
```

Comment:
i think this is wrong, because we still use `$contentRepositoryInstance->handle(CreateWorkspace::create(` below

Reply:
Right, the command controller needs some more love (and way less domain logic).
Good call, I'm on it!